packages/terra-framework-docs/src/terra-dev-site/test/worklist-data-grid/WorklistDataGridFocusableCell.test.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+import WorklistDataGrid from 'terra-worklist-data-grid';
+import NotificationDialog from 'terra-notification-dialog';
+
+const WorklistDataGridFocusableCell = () => {
+  const rowHeaderIndex = 0;
+
+  const [isOpen, setIsOpen] = useState(false);
+
+  const handleCloseModal = () => {
+    setIsOpen(false);
+  };
+
+  const handleButtonOpenModal = () => {
+    setIsOpen(true);
+  };
+
+  const buttonCell = <button type="button" aria-label="Button Element" onClick={handleButtonOpenModal}>Alert</button>;
+  // eslint-disable-next-line react/forbid-dom-props
+  const inputCell = <input type="text" aria-label="Text Input Element" style={{ width: '100px', height: '25px', display: 'inline' }} />;
+  const anchorCell = <a href="https://www.oracle.com/" aria-label="Anchor Element">Visit Oracle</a>;
+  const textAreaCell = <textarea name="textArea" aria-label="Text Area Element" readOnly rows="1" cols="15" value="Text Area" />;
```

Comment:
I'd assume since this has a name you wouldn't get a linter warning for not having an aria label but just in case
```suggestion
  const textAreaCell = <textarea name="textArea" aria-label="Text Area" readOnly rows="1" cols="15" value="Text Area" />;
```